src/tests/mockData.js

```diff
@@ -389,5 +389,28 @@ const mockData = {
     checkinDate: '2020-06-13',
     checkoutDate: '2020-06-24',
   },
+  superAdminLogin: {
+    email: 'superadmin@barefootnomad.com',
+    password: 'Niyonkuru@1',
+  },
+  regularStaffLogin: {
+    email: 'alain.maxime@gmail.com',
+    password: 'Niyonkuru@1',
+  },
+  regularStaffToken: {
+    token: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiZW1haWwiOiJhbGFpbi5tYXhpbWVAZ21haWwuY29tIiwicm9sZSI6InJlcXVlc3RlciIsImZpcnN0TmFtZSI6IkFsYWluIiwibGFzdE5hbWUiOiJNYXhpbSIsImlhdCI6MTU4NDQ2MTQzOX0.zjVLe__UqAqvjPCToCXuuZ64AV8o2rOeN-jYXx7K-AY',
+  },
+  updatableUser: {
+    email: 'brftnomad@gmail.com',
+  },
+  wrongUser: {
+    email: 'xxxxxx',
+  },
+  adminToken: {
+    adminRole: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6MSwiZW1haWwiOiJzdXBlcmFkbWluQGJhcmVmb290bm9tYWQuY29tIiwicm9sZSI6InN1cGVyQWRtaW4iLCJmaXJzdE5hbWUiOiJNb3NlIiwibGFzdE5hbWUiOiJSd2lidXRzbyIsImlhdCI6MTU4NDM4NjQwMX0.eELw_RvIbhDChs6X7fD3SwZ4yZo94PAIQ5jEk4we2P0',
```

Comment:
@Moise1  could you please not hard copy the token, this not safe rather find a way to login a user to get the token.